lib/smart_proxy_dns_infoblox/plugin_configuration.rb

```diff
@@ -9,7 +9,7 @@ def load_classes
     def load_dependency_injection_wirings(container_instance, settings)
       container_instance.dependency :connection,
                                     (lambda do
-                                      ::Infoblox.wapi_version = '2.0'
+                                      ::Infoblox.wapi_version = '1.4.2'
```

Comment:
Why did you revert to the older version of wapi?

Reply:
This is specific to our environment, where the infoblox is an old version not compatible with the 2.0 version presented.
To assure a maximum compatibility, the WAPI version should be set to the minimum version required with the verbs used (something like 1.0 or 1.2) and only pushed forward if the api used requires it.

Reply:
Considering limited amount of infoblox users, most of whom use WAPI version 2.0, and infoblox api gem using version 2.0 by default, I don't think it's a good idea to downgrade api version in this fashion.

Could you remove this change please? If you think it's important to have the ability to change api version, could you create a dedicated PR for this (probably could use a config parameter for this)?